analyzer/queries/queries.go

```diff
@@ -22,10 +22,10 @@ var (
       )
     )`
 
-	UnlockBlockForProcessing = `
+	UnlockBlocksForProcessing = `
     UPDATE analysis.processed_blocks
     SET locked_time = '-infinity'
-    WHERE analyzer = $1 AND height = $2`
+    WHERE analyzer = $1 AND height = ANY($2::uint63[])`
```

Comment:
:+1: nice side effect/feature. Done.
Efficiency shouldn't be a problem; we always limit ourselves to a small set of heights, so that should incentivize the db to use the height-based index.